plugins/importexport/native/NativeImportDom.inc.php

```diff
@@ -813,6 +813,10 @@ function handleArticleNode(&$journal, &$articleNode, &$issue, &$section, &$artic
 		}
 		if ($hasErrors) return false;
 		
+		$citationDao =& DAORegistry::getDAO('CitationDAO');
```

Comment:
Have you tested an import? The tool we use for continuous integration tests fails on the import test, and it's probably because of this section, I think.